.github/workflows/release.yml

```diff
@@ -33,24 +34,30 @@ jobs:
     needs: versioning
     strategy:
       matrix:
-        os: [ubuntu-20.04, macOS-11]
+        os: [ubuntu-20.04, macOS-11, windows-2022]
 
     steps:
       - uses: actions/checkout@v3
         with:
           ref: 'main'      
       - uses: actions/setup-python@v3
-      - uses: crazy-max/ghaction-setup-docker@v1.0.0
+      
+      - name: Setup Docker
+        if: matrix.os != 'windows-2022'
+        uses: crazy-max/ghaction-setup-docker@v1.0.0
         with:
           version: 23.0.1
```

Comment:
Suggest to update to latest `v1.4.0` (or use `v1`) and set `SIGN_QEMU_BINARY=1` env var per https://github.com/crazy-max/ghaction-setup-docker#macos-qemustderr-qemu-system-x86_64-unknown-error

```yaml
    - name: Setup docker
      if: matrix.os != 'windows-2022'
      uses: crazy-max/ghaction-setup-docker@v1
      env:
        SIGN_QEMU_BINARY: 1
```

Reply:
Sounds good! @crazy-max, can you please open a PR into the main-dev branch? I believe we may have the same issue in USearch, and maybe even a couple of personal projects, like StringZilla and SimSIMD.